src/main/java/dk/alexandra/fresco/lib/helper/builder/AbstractProtocolBuilder.java

```diff
@@ -67,6 +71,10 @@
 	 */
 	@Override
 	public void beginParScope() {
```

Comment:
Well yea I guess. You'd eliminate the return statement which, if forgotten, would be bad.